src/libraries/System.Security.Cryptography.Algorithms/src/Resources/Strings.resx

```diff
@@ -153,6 +153,9 @@
   <data name="Cryptography_HashAlgorithmNameNullOrEmpty" xml:space="preserve">
     <value>The hash algorithm name cannot be null or empty.</value>
   </data>
+  <data name="Cryptography_HashAlorithmNoSupportForGetCurrent" xml:space="preserve">
+    <value>The hash alorithm does not support get current.</value>
```

Comment:
I don't think this message is needed, since the only place that throws it is internal.

Reply:
Removed